tests/api_test/api_test.cpp

```diff
@@ -314,3 +314,83 @@ TEST_CASE("divide_by_zero_jit", "[divide_by_zero]") { divide_by_zero_test_km(EBP
 TEST_CASE("ringbuf_api_interpret", "[test_ringbuf_api]") { ring_buffer_api_test(EBPF_EXECUTION_INTERPRET); }
 TEST_CASE("divide_by_zero_interpret", "[divide_by_zero]") { divide_by_zero_test_km(EBPF_EXECUTION_INTERPRET); }
 #endif
+
+void
+_test_nested_maps(bpf_map_type type)
+{
+    // Create 2 inner maps.
+    fd_t inner1 = bpf_create_map(BPF_MAP_TYPE_ARRAY, sizeof(uint32_t), sizeof(uint32_t), 1, 0);
+    REQUIRE(inner1 > 0);
+
+    // Create outer map.
```

Comment:
consider moving these 2 lines above line 321.

Reply:
You can't since inner1 is used in line 326.

Reply:
Sorry, I missed that. However, couldn't you call the `create_map_in_map` API with fd to the template map that should be defined in the ELF file. Do we have a test for that?